src/plugins/render/nodes/draw.cpp

```diff
@@ -115,6 +117,8 @@ struct Drawable : public possumwood::Drawable {
 		const possumwood::VertexData& vertexData = values().get(a_vertexData);
 		possumwood::Uniforms uniforms = values().get(a_uniforms);
 
+		auto originalState = values().get(a_setup).apply();
```

Comment:
Codacy found an issue: [Variable 'originalState' is assigned a value that is never used.](https://app.codacy.com/manual/martin.prazak/possumwood/pullRequest?prid=6280524)